src/presentation/pages/Verticals/HallOfFame/HallOfFame.tsx

```diff
@@ -0,0 +1,152 @@
+import { Accordion, AccordionDetails, AccordionSummary, Button, Grid, Typography } from '@material-ui/core';
+import { Remove, Add } from '@material-ui/icons';
+import { CustomLoader } from 'infrastructure/components/CustomLoader/CustomLoader';
+import { Item } from '../Main/Item';
+import { useParams } from 'react-router';
+import { Promotion } from '../Promotion/Promotion';
+import { useHallOfFame } from './useHallOfFame';
+import VerticalPromotion from 'app/assets/VerticalPromotion.png';
+import useTranslation from 'app/hooks/useTranslation';
+import { NFT } from 'app/interfaces/NFT/NFT';
+import { LEGENDS, TIERS } from 'app/constants/contants';
+import MarcusAllen from 'app/assets/Athlete-MarcusAllen.png';
+import DoakWalker from 'app/assets/Athlete-DoakWalker.png';
+import EarlCampbell from 'app/assets/Athlete-EarlCampbell.png';
+import JoeTheissmann from 'app/assets/Athlete-JoeTheissmann.png';
+import ShannonSharpe from 'app/assets/Athlete-ShannonSharpe.png';
+import TimBrown from 'app/assets/Athlete-TimBrown.png';
+import MoreSoon from 'app/assets/Athlete-MoreSoon.png';
+import styles from './HallOfFame.module.scss';
+import mainContentStyles from '../Main/Main.module.scss';
+import { ItemBanner } from './ItemBanner/ItemBanner';
+
+export const HallOfFame = () => {
+  const { verticalId } = useParams<{ verticalId?: string }>();
+  const { nfts, isLoading, setTier, setLegend, legend, tier } = useHallOfFame();
+  const t = useTranslation();
+
+  return (
+    <Accordion className={styles.hallOfFame__accordion}>
+      <AccordionSummary>
+        <Promotion
+          imgSrc={VerticalPromotion}
+          subtitle={t('verticals.playbooks.title')}
+          primaryText={t('verticals.playbooks.primaryText')}
+          secondaryText={t('verticals.playbooks.secondaryText')}
+          buttonText={t('verticals.playbooks.actionButton')}
+        />
+      </AccordionSummary>
+      <AccordionDetails>
+        {isLoading ? (
+          <CustomLoader />
+        ) : (
+          <Grid container>
+            <Grid container justifyContent="center">
+              <ItemBanner
```

Comment:
can we create an array here and iterate over it?

Reply:
How do you recommend doing this one? I'm not sure how to approach this, taking into account that all has different images.